addrreg/models/__init__.py

```diff
@@ -132,7 +132,9 @@ class Meta(object):
         default='https://data.gl/najugaq/municipality',
     )
 
-    code = models.PositiveSmallIntegerField(_('Code'), db_index=True)
+    code = models.PositiveSmallIntegerField(
+        _('Code'), db_index=True, unique=True
+    )
```

Comment:
What's the motivation for this change?

Reply:
The test-case `def test_create_duplicate_municipality_fails(self):` seemed to expect uniqueness of the code field.

Reply:
It's better to just remove the test, and this change along with it.

Reply:
Alright.